appserver/admin/production_domain_template_web/src/main/resources/config/domain.xml

```diff
@@ -401,7 +406,9 @@
         <jvm-options>[9|]--add-opens=java.base/java.nio=ALL-UNNAMED</jvm-options>
```

Comment:
Following `jvm-options` missing in `default-config`

- java.base/java.net 
- java.base/java.util 
- java.rmi/sun.rmi.transport